src/EnergyPlus/DXCoils.cc

```diff
@@ -1230,13 +1231,17 @@ namespace DXCoils {
 				}
 			}
 
-			DXCoil( DXCoilNum ).Twet_Rated( 1 ) = Numbers( 6 );
-			DXCoil( DXCoilNum ).Gamma_Rated( 1 ) = Numbers( 7 );
-			DXCoil( DXCoilNum ).MaxONOFFCyclesperHour( 1 ) = Numbers( 8 );
-			DXCoil( DXCoilNum ).LatentCapacityTimeConstant( 1 ) = Numbers( 9 );
+			//Set minimum OAT for compressor operation
+			DXCoil( DXCoilNum ).MinOATCompressor = Numbers( 6 );
+			if ( NumNumbers < 6 ) DXCoil( DXCoilNum ).MinOATCompressor = minOATCompDXCooling; // input field is after min fields and won't default if field not included 
```

Comment:
No, I'm talking about the same field (field A in your example). Those fields that are past min fields have a problem with defaults in the IDD if the user does not enter this field (NumFields < this field number). The developer does not know what the default is if the user changes it and then doesn't enter that field as a blank in the object. I guess this is one reason we have min fields.

Instead of defaulting like this to -25C (hard coded), we could default to NumericFieldDefault(6) and then all defaults past min fields could be initialized in code and the user would still be able to change defaults in the IDD and not worry about entering a blank field (after min fields). Why should a user or GUI have to worry about these latter fields when there are defaults. MinFields should be just that, the very minimum.